README.md

```diff
@@ -124,6 +124,30 @@ This is particularly useful if you need to produce multiple aXe reports within
 the same test as it would otherwise keep replacing the same report every time
 you run the assertion.
 
+Two report types are supported: HTML and JSON. You can pass options for each
+separately:
+
+```js
+await expect(page.locator('#my-element')).toBeAccessible({
+  reports: {
+    html: {
+      // By default, this library attaches the HTML report on failure,
+      // but you can override this behaviour with attach='on' or 'off'
+      attach: 'retain-on-failure', // 'on' | 'off' | 'retain-on-failure'
```

Comment:
I'm not sure I like this key name, "attach"

Reply:
I'm not sure I'm a fan of this much structure.  I'd honestly like to keep this more simple with a single key for each report which could be a boolean or filename.  If you want to always print a report, I'd recommend building that custom, it doesn't fit the core use case of this library.